### PR TITLE
Add com.github.appadeia.Taigo

### PR DESCRIPTION
**Pull request by Appadeia**

### CODE REVIEW

**Comment by Appadeia** [comment]
bot, build me.appadeia.Taigo

**Comment by Appadeia** [comment]
bot, build me.appadeia.Taigo

**Comment by Appadeia** [comment]
bot, build me.appadeia.Taigo

**Comment by Appadeia** [comment]
bot, build me.appadeia.Taigo

**Comment by Appadeia** [comment]
bot, build me.appadeia.Taigo

**Comment by Appadeia** [comment]
bot, build me.appadeia.Taigo

File: me.appadeia.Taigo.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{
+    "app-id": "me.appadeia.Taigo",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.32",
```
Comment (LeandroStanger) [inline me.appadeia.Taigo.json]: Update runtime to 3.34

File: me.appadeia.Taigo.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    "command": "taigo",
+    "finish-args": [
+        "--share=network",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--filesystem=xdg-run/dconf",
+        "--filesystem=~/.config/dconf:ro",
+        "--talk-name=ca.desrt.dconf",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
```
Comment (gasinvein) [inline me.appadeia.Taigo.json]: This won't be needed with Gnome SDK 3.34+

File: me.appadeia.Taigo.json [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    ],
+    "cleanup": [
+        "/include",
+        "/lib/pkgconfig",
```
Comment (bilelmoussaoui) [inline me.appadeia.Taigo.json]: The cleanup section here is too generic.

File: me.appadeia.Taigo.json [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        "/share/man",
+        "/share/pkgconfig",
+        "/share/vala",
+        "*.la",
+        "*.a"
+    ],
+    "modules": [
+        {
+            "name": "taigo",
+            "builddir": true,
```
Comment (bilelmoussaoui) [inline me.appadeia.Taigo.json]:
```suggestion
```

**Comment by Appadeia** [comment]
bot, build me.appadeia.Taigo

**Comment by Appadeia** [comment]
bot, build me.appadeia.Taigo

**Comment by nedrichards** [comment]
The button in the play dialog is weirdly cut off, is that a bug or is this designed to only work with a certain theme?
Screenshot from 2019-11-26 20-50-57

do you own the appadeia.me domain? An app-id should be a domain you own or control see https://github.com/flathub/flathub/wiki/App-Requirements#Application-ID for more e.g. com.github.appadeia.Taigo

**Comment by Appadeia** [comment]
bot, build com.github.appadeia.Taigo

File: me.appadeia.Taigo.json [hunk 5]
```diff
@@ -0,0 +1,39 @@
+{
+    "app-id": "com.github.appadeia.Taigo",
```
Comment (bilelmoussaoui) [inline me.appadeia.Taigo.json]: The file needs to be renamed too.

**Comment by Appadeia** [comment]
> The button in the play dialog is weirdly cut off, is that a bug or is this designed to only work with a certain theme?
> Screenshot from 2019-11-26 20-50-57

Is that HiDPI?

**Comment by Appadeia** [comment]
bot, build com.github.appadeia.Taigo

**Comment by nedrichards** [comment]
@Appadeia yeah, it's hidpi.

**Comment by Appadeia** [comment]
> @Appadeia yeah, it's hidpi.

I'll look into fixing high DPI support in a future release of Taigo.

**Pull request closed without merging**